chef/cookbooks/crowbar/templates/default/crowbar-delete-pacemaker-resources.sh.erb

```diff
@@ -29,6 +29,14 @@ rm -f $UPGRADEDIR/crowbar-delete-pacemaker-resources-failed
 
 <% if @use_ha %>
 
+log "Resetting rabbitmq cluster..."
+# TODO: remove if not needed
+#rabbitmq_nodes=$(rabbitmqctl cluster_status | tr -d '\n' | sed -r -e 's/\s//g' -e 's/.*\{disc,\[//' -e 's/\].*//' -e "s/'//g" -e 's/,/ /g')
+for node in <% @rabbit_nodes.join(" ") %>; do
```

Comment:
There could still be a check for the rabbit resource by pacemaker between this reset and the resource stop down in line 48 which could lead to rabbitmq resource being restarted.

Not sure if that could lead to unintended consequences, just wanted to mention that.

Reply:
That actually sounds important. We should not just kill services handled by pacemaker, otherwise pacemaker will revive them.

Why doesn't the pacemaker RA do the right cleanup when stopping?

Reply:
If this turns out to be a problem we can try to reset the nodes after stopping pacemaker resources the way you suggested before (i.e. start standalone rabbit + reset + stop).

Reply:
So maybe we need to edit the rabbitmq resource and set (both?) the monitor intervals to 0? And then reset?

Reply:
@jsuchome about killing services, this is a little tricky with rabbit: we just stop the app without stopping the erlang vm, not sure how pacemaker will react to that but idea from @matthewoliver should take care of this too I guess...

Reply:
pacemaker will see that the rabbit app is not running as the agent checks for that explicitly[0] and will restart the service, but setting the monitor interval to 0 sounds great as to avoid it triggering anything while we do this

[0] https://github.com/rabbitmq/rabbitmq-server-release/blob/master/scripts/rabbitmq-server-ha.ocf#L1554